extensions/amp-user-notification/0.1/test/test-amp-user-notification.js

```diff
@@ -27,14 +30,12 @@ describes.realWin('amp-user-notification', {
     extensions: ['amp-user-notification'],
   },
 }, env => {
-  let sandbox;
```

Comment:
Is using the `global.sandbox` a best practice for AMP tests that we should try to use elsewhere?